outputs.tf

```
@@ -8,6 +8,11 @@ output "flowlog_log_group_arn" {
   value       = join(" ", aws_cloudwatch_log_group.flowlog_group.*.arn)
 }
 
+output "flowlog_log_group_name" {
+  description = "The name of the flow log CloudWatch log group if one was created"
+  value       = aws_cloudwatch_log_group.flowlog_group.name
```

Hey @dplflorian,

Looks like I spoke too soon.  Since there is a count associated with this log group, we will need to use an index to access it.  Instead, maybe we could try something like:
```suggestion
  value       = var.build_flow_logs ? aws_cloudwatch_log_group.flowlog_group[0].name : ""
```

---

thanks @johnctitus. I haven't considered this.